client/components/shipping-rates/index.js

```diff
@@ -0,0 +1,54 @@
+import React, { PropTypes } from 'react';
+import Dropdown from 'components/dropdown';
+
+const ShippingRates = ( { selectedRates, availableRates, packages, updateValue, dimensionUnit, weightUnit, currencySymbol, errors, layout } ) => {
+	const renderTitle = ( pckg ) => {
+		const dimensions = [ pckg.length, pckg.width, pckg.height ].map( dim => dim + ' ' + dimensionUnit ).join( ' x ' );
+		const weight = pckg.weight + ' ' + weightUnit;
+		return <span>{ dimensions + ' ' + weight }</span>;
```

Comment:
I went ahead and changed the strings to use ES6 templates. At this moment this string isn't going to change for any language ("W x L x H" is universal), so there's no need to localize it.